### PR TITLE
Cold storage Integration

### PR DESCRIPTION
**Pull request by YasseinBilal**

### CODE REVIEW

File: src/module/cold-storage/installation.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+export const getInstallColdStorageHook = ({
+  waitPeriod,
+  owner,
+  moduleType,
+}: Params): Module => {
+  return {
+    module: COLD_STORAGE_HOOK_ADDRESS,
+    data: encodePacked(['uint128', 'address'], [BigInt(waitPeriod), owner]),
```
Comment (kopy-kat) [inline src/module/cold-storage/installation.ts:21]: the data should be empty if module is already installed

File: src/module/cold-storage/installation.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    type: moduleType,
+  }
+}
+
+type FlashloanParams = {
+  addresses: Address[]
+}
+
+export const getInstallColdStorageFlashloan = ({
+  addresses,
```
Comment (kopy-kat) [inline src/module/cold-storage/installation.ts:32]: maybe lets rename this to whitelist

Comment (kopy-kat) [reply src/module/cold-storage/installation.ts:32]: or allowedCallbackSenders might be more descriptive

File: src/module/cold-storage/usage.ts [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    value: BigInt(0),
+    callData: encodeFunctionData({
+      functionName: 'addAddress',
+      abi: flashloanAbi,
+      args: [addressToAdd],
+    }),
+  }
+}
+
+export const getRemoveAddressAction = async ({
```
Comment (kopy-kat) [inline src/module/cold-storage/usage.ts:135]: I like using action instead of execution but should we port everything over?

Comment (YasseinBilal) [reply src/module/cold-storage/usage.ts:135]: @kopy-kat Yeah, I like action also more .. I could create a separate PR for that once done with integrating all modules

Comment (kopy-kat) [reply src/module/cold-storage/usage.ts:135]: sounds good

**Pull request merged**